.github/CODEOWNERS

```diff
@@ -27,6 +27,7 @@ x-pack/plugins/alerting @elastic/response-ops
 x-pack/packages/kbn-alerting-state-types @elastic/response-ops
 packages/kbn-alerting-types @elastic/response-ops
 packages/kbn-alerts-as-data-utils @elastic/response-ops
+packages/kbn-alerts-grouping @elastic/response-ops
```

Comment:
I prefer it to be renamed pure `kbn-grouping`, because we a grouping not only alerts, but other types of the data.

Reply:
++